### PR TITLE
Edit UserGuide to make it more user-specific and friendly

### PR DESCRIPTION
**Pull request by cheahTJ**

### CODE REVIEW

**Comment by cheahTJ** [comment]
Resolved #214 #210 #199 #166 #163

File: docs/UserGuide.md [hunk 1]
```diff
@@ -4,11 +4,19 @@
...
+
+Welcome to AronaPro's User Guide, and thank you for choosing us as your all-in-one solution! 
+Our product is a desktop app designed primarily for **Computing Professors and Teaching Assistants(TAs)** to manage your 
+students/professors/TAs' contacts and schedules efficiently, optimised for use via a Command Line Interface (CLI) 
+while still having the benefits of a Graphical User Interface (GUI). Our simple-to-grasp software also warmly welcomes
+professors and TAs from various faculties without any technological background! 
+
+We are excited to have you on board and to guide you through how to use our software with this guide.
 
+Together, let's **_Centralise_, _Connect_ and _Coordinate_!**
```
Comment (hjuntan) [inline docs/UserGuide.md:19]: ALLITERATION MENTIONED OWWO

**Review by hjuntan — COMMENTED** [review]
Can you adjust the issue mentioned in #198 ; Changing from `t/TAG` to `g/GROUP`: intended features

File: docs/UserGuide.md (hunk 1 above)
Comment (cheahTJ) [reply docs/UserGuide.md:19]: Thank you for the good catch, on it!

**Comment by hjuntan** [comment]
Good changes to the UG that can be used in CS2101!

**Pull request merged**